Sources_v3/StreamChat/WebSocketClient/WebSocketClient_Tests.swift

```diff
@@ -273,9 +318,11 @@ class WebSocketClient_Tests: StressTestCase {
     }
     
     func test_webSocketPingController_ping_callsEngineWithPing() {
-        assert(engine.sendPing_calledCount == 0)
```

Comment:
This assert was important. We need to make sure the `sendPing_calledCount` is `0` when we call `sendPing()`. We can reset the counter manually after `test_connectionFlow()` like `engine!.sendPing_calledCount = 0`. Same thing for all other deleted asserts in this commit.

Reply:
fixed in c732d658ca9625bd74d35c32c4d80b3a70871a0e and also used force-unwrap for `engine` when we expect it to exist